arangod/Pregel/Worker.cpp

```diff
@@ -446,7 +459,8 @@ bool Worker<V, E, M>::_processVertices(size_t threadId,
     _messageStats.accumulate(stats);
     _activeCount += activeCount;
     _runningThreads--;
-    lastThread = _runningThreads == 0;  // should work like a join operation    
+    _reports.append(std::move(reports));
```

Comment:
`reports` is a local variable, to which then `vertexComputation` gets a pointer, fills it (I assume), and then `reports` is moved away. That is a little confusing to me. It seems simpler for a `VertexComputation` just to have its own `Report` which could then be moved later. Or does that have other impacts I'm not seeing yet?